tests/garage/envs/mujoco/randomization/test_dynamics_rand.py

```diff
@@ -31,8 +40,27 @@ def setUp(self):
         body_id = self._env.sim.model._body_name2id[self._bodyname]
         self._orig_val = np.array(self._env.sim.model.body_pos[body_id])
 
+    def create_randomized_xml(self):
+        with io.StringIO() as model_xml:
+            self._env.sim.save(model_xml, 'xml')
```

Comment:
Mujoco writes a temporary file that is not closed after saving it to `model_xml`. I get warnings like this

> ResourceWarning: unclosed file <_io.TextIOWrapper name='/tmp/tmpgir8q4j2/model.xml' mode='r' encoding='UTF-8'>

Reply:
Would this be simpler if you just loaded the XML from its source instead of trying to dynamically get it by asking the environment?